assets/js/main.js

```diff
@@ -49,7 +49,37 @@
       });
     });
   });
+// //login/signup
+const formOpenBtn=document.querySelector("#form-open"),
+home =document.querySelector(".home"),
+formContainer=document.querySelector("form-container"),
+formCloseBtn=document.querySelector(".form_close"),
+signupBtn=document.querySelector("#signup"),
+loginBtn=document.querySelector("#login"),
+pwShowHide=document.querySelectorAll(".pw_hide");
 
+formOpenBtn.addEventListener("click",()=>home.classList.add("show"));

```

Comment:
instead of this directly use 
```js
formOpenBtn.addEventListener("click",()=>home.style.display = 'none');
```
```js
formCloseBtn.addEventListener("click",()=>home.style.display = 'block');
```